.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
   - OPAMYES="true"
   - TEST_TARGETS="unit_tests real_tests"
   matrix:
-  - OCAML_VERSION=4.04.1
+#  - OCAML_VERSION=4.04.1
```

Comment:
What is the problem with `4.04.1`?

Reply:
From the travis log,
> ```$ make $TEST_TARGETS
> 
> File "/home/travis/.opam/4.04.1/lib/ppx_inline_test/ppx_inline_test.dune", line 1, characters 0-0:
> 
> Warning: .dune files are ignored since 2.0. Reinstall the library with dune
> 
> >= 2.0 to get rid of this warning and enable support for the subsystem this
> 
> library provides.
> 
> File "src/pure/dune", line 4, characters 1-15:
> 
> 4 |  (inline_tests)
> 
>      ^^^^^^^^^^^^^^
> 
> Error: No inline tests backend found.
> ```

Reply:
I think this means we need a constraint on the dune version.

Reply:
It looks like Ocaml 4.04.1 needs `dune <= 2.0` (since 2.3 is installed and it does not work). Can we constrain on only one version of OCaml?

Reply:
What is weird is that it seems that the inline test thingy is not installed properly. I'm guessing this is an Opam or Dune bug in fact.

Reply:
Can you replicate the error on your machine ?

Reply:
I can

Reply:
And can you actually understand what happens?

Reply:
At least it works if I install dune 1.11.4 instead of 2+

Reply:
I guess `ppx_inline_test` version `0.11.0` should constrain `dune` to be `<= 1.11.4`

Reply:
OK, maybe you can report that upstream? Anyway let's just keep what you did, #289 actually removes the use of `ppx_inline_test`.